scripts/stac/imagery/collection.py

```diff
@@ -32,6 +32,7 @@
 from scripts.stac.util.stac_extensions import StacExtensions
 
 CAPTURE_AREA_FILE_NAME = "capture-area.geojson"
+CAPTURE_DATES_FILE_NAME = "capture-dates.geojson.gz"
```

Comment:
I thought we were ungzipping this one?

Reply:
I thought it was agreed to keep it gzipped... :laughing:

Reply:
I just ran it again, it generated a 3.8MB geojson, Ideally this should be `.geojson` but then when transfered/downloaded/stored it should be compressed, but due to being a s3 location I dont think we can easily set that across all the tools (argo + everything else?)

Reply:
Decision made not to gzip the capture dates file 👍